### PR TITLE
Built the Real Dev Squad Members section

### PR DESCRIPTION
**Pull request by NabajitS**

### CODE REVIEW

**Comment by NabajitS** [comment]
Issue #21

Feature Details:
Built the RealDevSquad Members Section for members-site referencing the previous members section

**Reference(previous):**

Screenshot 2023-03-02 170731

After change screenshot

**Desktop View**

Screenshot 2023-03-02 170048

**Mobile View**

Screenshot 2023-03-02 170145

File: README.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ You can find a detailed guide [here](./CONTRIBUTING.md)!
...
 ### There are only 2 things in computer science: cache invalidation & naming things  — *********Phil Karlton*********
 
+## Imports Order 
+- External Libraries 
+- Internal Libraries
+- Components 
+- Types 
+- Images 
+- CSS 
+
```
Comment (vinayak-trivedi) [inline README.md:24]: why is this changes?

File: src/components/Members Section/MemberCard.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+const MemberCard: FC<MemberProps> = ({user}) =>  {
+  return (
+    <div className='m-4 pb-3 text-center shrink shadow-cus grow max-w-cusXS border border-gray-800 rounded-2xl transform 
+     transition duration-500 hover:scale-105 ' >
+    
+    {/* Member Image */}
+    <div className='m-auto w-48 h-48 mt-4' >
+      <img className='w-full h-full rounded-full' src={user.image} alt="randomS" />
```
Comment (vinayak-trivedi) [inline src/components/Members Section/MemberCard.tsx:22]: use next Image

File: src/components/Members Section/MemberCard.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            <a target="_blank" href={user.twitter}>
+              <img className='h-8' src="/icons/instagram.png" alt="instagram icon" />
+            </a>
+        )
+       }
+
+
+    </div>
+
+    </div>
```
Comment (vinayak-trivedi) [inline src/components/Members Section/MemberCard.tsx:65]: can you fix indentation here and other places as well?

File: src/components/Members Section/MemberCard.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      
+    <h1 className='text-2xl m-1 font-semibold' >{user.name}</h1>  {/* Member Name  */}
+
+    {/* Member Social Links */}
+    <div className='flex justify-center gap-2.5 my-3' > 
+      {
+        user.twitter && (
+            <a target="_blank" href={user.twitter}>
+              <img className='h-8' src="/icons/twitter.png" alt="twitter icon" />
+            </a>
```
Comment (vinayak-trivedi) [inline src/components/Members Section/MemberCard.tsx:33]: you can use next Image and Link

**Review by vinayak-trivedi — CHANGES_REQUESTED** [review]
Thanks for the PR @NabajitS, suggested some changes please look at them, also please use svgs instead of png

**Comment by NabajitS** [comment]
Done with the said changes

File: README.md (hunk 1 above)
Comment (NabajitS) [reply README.md:24]: IG this is showing up from someone else's commit as it was not in my initial PR

Comment (vinayak-trivedi) [reply README.md:24]: still this should not be there right?

**Pull request closed without merging**